platform_main_feature.info

```diff
@@ -105,20 +102,12 @@ features[fe_block_settings][] = views-image_blocks-block_2
 features[fe_block_settings][] = views-object_page_image-block
 features[fe_block_settings][] = views-object_subtitle-block
 features[fe_block_settings][] = views-object_visitors_by_month-block
-features[fe_block_settings][] = views-owl_carousel_view-block
 features[fe_block_settings][] = views-plaque_view-block
 features[fe_block_settings][] = views-social_media-block
 features[fe_block_settings][] = views-thumb_details-block
 features[fe_block_settings][] = views-top_facet_searches-block
 features[fe_block_settings][] = views-top_site_searches-block
 features[fe_block_settings][] = views-useful_information-block
-features[fe_block_settings][] = widgets-s_addthis-share
-features[fe_block_settings][] = widgets-s_facebook-like-box
-features[fe_block_settings][] = widgets-s_socialmedia_profile-buttons
```

Comment:
Seems to still be there, among others being removed here: https://github.com/discoverygarden/platform_main_feature/blob/792371a659daa45e17336c2cc5361abffcdf722f/platform_main_feature.features.fe_block_settings.inc#L2465-L2496